### PR TITLE
Fork Stuff

### PR DESCRIPTION
**Pull request by jrddunbr**
This is a bit of information relevant to becoming a fork.

It's related to #77 and #64 

I also want to fix Travis builds and do some refactoring to the contact information, status checks, analytics, etc.

This is *not complete* yet. I have a few things to add/fix/remove.

Sorry lashtear, I didn't dig through the commit history. I also have other contributors where I need to go back and look, this was just a brief first pass.

### CODE REVIEW

File: src/main/resources/mcmod.info [hunk 1]
```diff
@@ -3,12 +3,12 @@
         "modid":"Eln",
         "version": "${version}",
         "mcversion": "${mcversion}",
-        "name":"Electrical Age",
-        "url":"http://electrical-age.net/",
+        "name":"Electrical Age - jrddunbr edition",
```
Comment (Grissess) [inline src/main/resources/mcmod.info]: I can agree with almost everything else you've changed, but this seems a little excessive--I think leaving the name would be fine :)
```suggestion
        "name":"Electrical Age",
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
-[![Build Status](https://travis-ci.org/Electrical-Age/ElectricalAge.svg?branch=1.7.10-MNA)](https://travis-ci.org/Electrical-Age/ElectricalAge)
-[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FElectrical-Age%2FElectricalAge.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2FElectrical-Age%2FElectricalAge?ref=badge_shield)
-
 Electrical Age (ELN) is a Minecraft Mod offering the ability to perform large-scale in-game electrical simulations.
 
-Look at the official project website [electrical-age.net](https://electrical-age.net/) and [the Wiki](http://wiki.electrical-age.net/) to get general information. [This official Minecraft forum post](http://www.minecraftforum.net/topic/2741783-172forge-electrical-age-mod-beta-146/) is updated frequently.
+Look at the official project website [electrical-age.net](https://electrical-age.net/) and [the Wiki](http://wiki.electrical-age.net/) to get general information.
+
+This is a fork of Electrical Age, maintaine by Jared Dunbar (jrddunbr) and Graham Nothup (Grissess), with the intent to add features and push for active development, which is funneled back into the main distribution.
```
Comment (Grissess) [inline README.md]: "funneled" is awkward; maybe "incorporated"?
```suggestion
This is a fork of Electrical Age, maintaine by Jared Dunbar (jrddunbr) and Graham Nothup (Grissess), with the intent to add features and push for active development, which is incorporated back into the [main distribution](https://github.com/Electrical-Age/ElectricalAge).
```

File: src/main/resources/mcmod.info (hunk 1 above)
Comment (jrddunbr) [reply src/main/resources/mcmod.info]: Maybe `jrddunbr's-build`?

File: README.md (hunk 2 above)
Comment (jrddunbr) [reply README.md]: returned?

**Comment by jrddunbr** [comment]
Alright, merging now.

**Pull request merged**